public/js/clients/firefox.js

```diff
@@ -49,10 +50,13 @@ function createTabs(tabs) {
 function connectClient() {
   const deferred = defer();
   let isConnected = false;
-  const webSocketPort = getValue("firefox.webSocketPort");
+  const useProxy = !getValue("firefox.webSocketConnection");
```

Comment:
Well, the `getValue` symbol is already imported in the file. I don't know why it has two aliases :)